docs/catbytes_api_swagger.yaml

```diff
@@ -1150,4 +1152,58 @@ paths:
         '403':
           description: You don't have the rights to edit this mentor
         '404':
-          description: Mentor not found
\ No newline at end of file
+          description: Mentor not found
+  /generate-invite:
+    post:
+      tags:
+        - discord integration
+      summary: Generate a Discord invite link
+      description: |
+        Generates a unique invite link for a user to join the Discord server. 
+        Validates if the user already exists on the server and checks if 7 days have passed since the last invite.
+        Requires authentication via a `userUID` cookie.
+      responses:
+        '200':
+          description: Invite link generated successfully
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  inviteLink:
+                    type: string
+                    description: The generated Discord invite link.
+                    example: "https://discord.gg/abc123"
+        '400':
+          description: "User already exists on the server or invite generation is on cooldown for this user"
```

Comment:
what does "is on cooldown" mean? 

also, I think it'd be better self-descriptive if we returned 403 on "user reached limit of invitations"